src/rust/sysmon-subgraph-generator/src/main.rs

```diff
@@ -145,13 +145,20 @@ fn handle_process_start(process_start: &ProcessCreateEvent) -> Result<Graph, Err
         child.clone_node_key(),
     );
 
+    graph.add_edge(
+        "process_asset",
+        child.clone_node_key(),
+        asset.clone_node_key(),
+    );
+
     graph.add_edge(
         "bin_file",
         child.clone_node_key(),
         child_exe.clone_node_key(),
     );
 
     graph.add_edge("children", parent.clone_node_key(), child.clone_node_key());
+    graph.add_edge("parent", child.clone_node_key(), parent.clone_node_key());
```

Comment:
these edge names seem a little to generic imo - i think i'd prefer 'parent_process' 'child_process'. "children" plural seems odd too, considering it's for a single edge, but i guess i can see wanting to use it in query `with_children`. not sure.

Reply:
I agree. We should take some time to do a final pass through our schemas.